src/main/resources/db/migration/V3_0__fix_melding_type_innkommende_legeerklaring.sql

```diff
@@ -0,0 +1,30 @@
+UPDATE melding SET type='FORESPORSEL_PASIENT_LEGEERKLARING' WHERE
+innkommende AND type='FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER'
```

Comment:
Disse to kriteriene er egentlige overflødige, men gjør det kanskje mulig å skjønne hva som er poenget.